_ingest-pipelines/processors/foreach.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: Foreach
+parent: Ingest processors
+nav_order: 110
+---
+
+# Foreach processor
+
+The `foreach` processor is used to <explain what is used to do>.
+
+The following is the syntax for the `foreach` processor:
+
+```json
+<insert syntax example>
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `foreach` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+<insert the parameters>
+
```

Comment:
@gaobinlong Please confirm the configuration parameters table is correct for this processor.

Reply:
The common parameter `description` is missing, and could we make the description of all common parameters same among all processors? See https://opensearch.org/docs/latest/ingest-pipelines/processors/rename/.